src/app/authentication/models/signUpModel.go

```diff
@@ -4,11 +4,19 @@ type UserInfo struct {
 	ID            int32  `json:"user_id"`
 	Name          string `json:"user_name"`
 	Email         string `json:"user_email"`
-	PhoneNumber   int32  `json:"user_phoneNumber"`
-	PanCardNumber int32  `json:"user_panCardNumber"`
+	PhoneNumber   string `json:"user_phoneNumber"` // Changed type to string
+	PanCardNumber string `json:"user_panCardNumber"`
 	Password      string `json:"user_password"`
```

Comment:
Keep all json name as camelCase as best practice and also add all possible in built and custom Validation.
Refer https://pkg.go.dev/github.com/go-playground/validator/v10#section-readme